### PR TITLE
[SINT-5370] Updated Python SDK v2 syntax and runtime to Python3.9+

### PR DESCRIPTION
**Pull request by marcos-vivar**
# TL;DR
- Deprecated TS Python SDK v2 support for Python 2
- Deprecated TS Python SDK v2 support for Python 3.7
- Upgraded TS Python SDK v2 support for Python3.9
- Upgraded TS Python SDK v2 support for Python3.10

# What?
Several projects are depending on TS Legacy SDKs and they needed to be upgraded so the systems they support can be safely updated as well.

### CODE REVIEW

File: trustar2/models/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from __future__ import absolute_import
+
```
Comment (pmodi-crest) [inline trustar2/models/__init__.py:1]: NIT: Can remove empty line before imports

**Review by pmodi-crest — APPROVED** [review]
Looks good!

File: tests/unit/handlers/test_account.py [hunk 2]
```diff
@@ -28,5 +28,6 @@ def test_ping_successfully(account, mocked_request):
 def test_get_enclaves_successfully(account, mocked_request, enclaves_response):
     mocked_request.get(URL + "/enclaves", json=enclaves_response)
     response = account.get_enclaves()
+    current_enclaves = [e.serialize() for e in response.data]
     assert response.status_code == 200
-    assert [e.serialize() for e in response.data] == enclaves_response
+    assert len(current_enclaves) == len(enclaves_response)
```
Comment (mihirvala-crest) [inline tests/unit/handlers/test_account.py:33]: hey @marcos-vivar,
Just wondering, is this change intentional as we are changing assertion ?

File: tests/unit/handlers/test_safelist.py [hunk 3]
```diff
@@ -23,32 +21,37 @@ def safelist():
...
     return result
 
 
 def test_get_safelist_summaries(mocked_request, safelist, safelist_summaries_json):
-    mocked_request.get(SAFELIST_SUMMARIES, status_code=200, json=safelist_summaries_json)
+    mocked_request.get(SAFELIST_SUMMARIES_URL, status_code=200, json=safelist_summaries_json)
     response = safelist.get_safelist_libraries()
-    assert [s.serialize() for s in response.data] == safelist_summaries_json
+    current_safelists = [s.serialize() for s in response.data]
+    assert len(current_safelists) == len(safelist_summaries_json)
```
Comment (mihirvala-crest) [inline tests/unit/handlers/test_safelist.py:44]: I think we can refactor assertion here as well

File: tests/unit/models/test_prioritized_indicator.py [hunk 4]
```diff
@@ -87,7 +87,8 @@ def test_prioritized_indicator_deserialization(prioritized_indicator_json):
 
 
 def test_prioritized_indicator_serialization(prioritized_indicator_obj, prioritized_indicator_json):
-    assert prioritized_indicator_obj.serialize() == prioritized_indicator_json
+    current_prioritized_indicator = prioritized_indicator_obj.serialize()
+    assert len(current_prioritized_indicator) == len(prioritized_indicator_json)
```
Comment (mihirvala-crest) [inline tests/unit/models/test_prioritized_indicator.py:91]: Same here. need to keep original assertion as well, I guess.

File: tests/unit/handlers/test_account.py (hunk 2 above)
Comment (marcos-vivar) [reply tests/unit/handlers/test_account.py:34]: Hey @hetsplunk 👋🏽 

Yes, we implemented this upgrade a while back https://github.com/trustar/trustar-sdk2-proto/pull/68 and we haven't updated defaults across models, but they have just been upgraded to serialize and deserialize without `None` values.

I put all there tests back and kept new ones! 👍🏽

Comment (mihirvala-crest) [reply tests/unit/handlers/test_account.py:34]: Understood!

**Review by mihirvala-crest — APPROVED** [review]
LGTM 🚀

**Review by dsolmirano — APPROVED** [review]
LGTM! Thanks, great job
